packs/data/fall-of-plaguestone.db/the-behemoth.json

```diff
@@ -314,7 +311,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The monster makes a melee Strike and, if the attack hits, it can push the creature back 10 feet (20 feet on a critical hit). The target must succeed at a @Check[type:fortitude|dc:19] save or be knocked @Compendium[pf2e.conditionitems.Prone]{Prone} by this attack.</p>"
+                    "value": "<p>The behemoth makes a melee Strike and, if the attack hits, it can push the creature back 10 feet (20 feet on a critical hit). The target must succeed at a @Check[type:fortitude|dc:19|showDC:gm] save or be knocked @Compendium[pf2e.conditionitems.Prone]{Prone} by this attack.</p>"
```

Comment:
```suggestion
                    "value": "<p>The behemoth makes a melee Strike and, if the attack hits, it can push the creature back 10 feet (20 feet on a critical hit). The target must succeed at a @Check[type:fortitude|dc:19] save or be knocked @Compendium[pf2e.conditionitems.Prone]{Prone} by this attack.</p>"
```